src/app/story/storysidebar/story.sidebar.component.spec.ts

```diff
@@ -57,7 +59,7 @@ describe('StorysidebarComponent testing', () => {
     expect(titleAnchorEl.href).toBe('https://bugzilla.redhat.com/show_bug.cgi?id=23456');
     const titleEl = fixture.debugElement.query(By.css('.sidebar__external-system-link > h2')).nativeElement;
     expect(titleEl.textContent).toBe('RHBZ#23456');
-    const sidebarPropertiesEl = fixture.debugElement.query(By.css('.sidebar-properties > tbody')).nativeElement;
+    const sidebarPropertiesEl = fixture.debugElement.query(By.css('.sidebar > .sidebar-properties > tbody')).nativeElement;
```

Comment:
Why is this change necessary?

Reply:
I think this was necessary before when I was doing 'inner-sidebar' but now that that's gone it should be removed.